include/fastdds/dds/core/policy/QosPolicies.hpp

```diff
@@ -53,13 +53,20 @@ class QosPolicy
     {
     }
 
-    QosPolicy(
+    explicit QosPolicy(
             bool send_always)
         : hasChanged(false)
         , send_always_(send_always)
     {
     }
 
+    QosPolicy(
```

Comment:
Declare this as `= default`, and add copy assignment operator (also with `= default`)